app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -272,6 +276,19 @@ class AttendeeFragment : Fragment() {
                 redirectToLogin()
             }
 
+            //Custom forms fetch
```

Comment:
either remove or add a meaningful comment. the method name is self explanatory IMO

Reply:
Will remove it